server/slack-bot.js

```diff
@@ -41,14 +41,26 @@ This bot demonstrates many of the core features of Botkit:
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
 require('isomorphic-fetch');
 const queryString = require('query-string');
+const port = process.env.PORT || process.env.VCAP_APP_PORT || 3000;
 const slackBotToken = process.env.SLACK_BOT_TOKEN || 
                       process.env.VCAP_APP_SLACK_BOT_TOKEN;
-
-if (!slackBotToken) {
+const slackBotHostRoute = process.env.VCAP_APP_SLACK_BOT_HOST_ROUTE;
+var host_url;
+
+if (slackBotToken) {
+  if (slackBotHostRoute) {
+    host_url = 'https://' + slackBotHostRoute;
```

Comment:
I think these are fairly consistent... was trying to make it easy to copy and paste whats presented on the bluemix page. To get the full URL, it would require a few more instructions.